[Bulk] Validation: Fixed issues in ODatadocs (#219)
MicrosoftDocs/OData-docs: Odata-docs/connectedservice/generating-multiple-files.md

---
title: "Generating multiple files"

> description: By default, the OData Connected Service stores all the generated code into a single file. However, you can opt to have the generated code split into multiple files

Reviewer: ```suggestion
description: Describes how to split the generated code into multiple files.
```
